scipy/stats/tests/test_mstats_basic.py

```diff
@@ -410,6 +410,12 @@ def test_trim_old(self):
         assert_equal(mstats.trimboth(x).count(), 60)
         assert_equal(mstats.trimtail(x).count(), 80)
 
+    def test_trimr(self):
+        x = ma.arange(10)
+        result = mstats.trimr(x, limits=(0.15, 0.14), inclusive=(False, False))
+        expected = [None, None, 2, 3, 4, 5, 6, 7, 8, None]
+        assert_equal(result, expected)
```

Comment:
`result` is a masked array, apparently the comparison in `assert_equal` with the list works. I did not check in the other Tests: is that the standard way to write the test cases for masked arrays?

Reply:
I follow the format in `test_trim` to write the present test.
`assert_equal` is stricter than other test utilities such as `assert_almost_equal`.

Reply:
iirc numpy test functions had stuff for masked array handling

Reply:
Actually, not exactly:
```
>>> x = np.ma.array([1,2,3,4,5], dtype=int, mask=[0,0,0,1,0])
>>> numpy.testing.assert_equal(x, [1,2,3,None,5])
>>> numpy.testing.assert_equal(x, [1,2,3,4,5])
>>> numpy.testing.assert_equal(x, [1,2,3,999,5])
```
It won't check that the mask is correct, so this test should check the mask separately.